cmd/main.go

```diff
@@ -189,22 +192,36 @@ func (c Configuration) consumers(db *pkg.AgentDB, genericDiffPaths *[]string) (c
 	return consumers
 }
 
+// Gets list of regexp objects from regexp paths
+func (c Configuration) compileRegex(listofPaths []string) []*regexp.Regexp {
```

Comment:
no need to make it method of Configuration, it is just a simple function
there is no usage of any of Configuration fields here

Reply:
I would just pass just the logger as an additional argument then, but not critical